src/ir_def/check_ir.ml

```diff
@@ -364,7 +364,15 @@ let store_typ t  =
 let rec check_exp env (exp:Ir.exp) : unit =
   (* helpers *)
   let check p = check env exp.at p in
-  let (<:) t1 t2 = check_sub env exp.at t1 t2 in
+  let (<:) t1 t2 =
+(*  try *)
```

Comment:
remove debug code?

Reply:
Well, it a pain to reinvent this everytime I need to track down a bug so I'd prefer to just comment it out...